images/github-runner/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:latest
+
+
+# system update
+RUN set -eux ; \
+    apt update ; \
+    apt -y upgrade ; \
+    apt -y install \
+            curl \
+            libicu-dev  \
+            build-essential \
+            git \
+            cmake \
+            python3-pip \
+            wget \
+            language-pack-en \
+            libcurl4-openssl-dev;
+
+# cuda install
+# https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html
+ENV DEBIAN_FRONTEND=noninteractive
+RUN set -eux ; \
+    wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-keyring_1.1-1_all.deb ; \
```

Comment:
Maybe here we should use the official nvidia image, but I prefer this approach to control the bench environment.

Reply:
I don't know if it have an impact, but if it works then let's keep it this way.

AFAIK, installing this way will install normal CUDA library. The official nvidia image in the other hand comes with "stub" library, meaning it has no code inside `.so` files. These file will then be populated by the `nvidia` runtime option when creating the container, so in theory the 2 approach don't have much differences.